### ISSUE TITLE
esModuleInterop error when using Typescript

### ISSUE THREAD

**Issue by junalmeida** [23/resumable.js#547]
This is the exception I am facing when using typescript:

```
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.ts(2497)
```

**Comment by junalmeida** [comment]
Update: This might be already solved on version `1.1.2` however this version is not yet available on npm.

**Comment by RobDaPraia** [comment]
Problem is in the resumable.d.ts, almost the last line:
export = Resumable.Resumable;

have a look at the latest version: https://github.com/23/resumable.js/blob/master/resumable.d.ts
export = Resumable;

You can update the file under \node_modules\resumablejs\resumable.d.ts

But probably this manual change will be overwritten when doing a "npm install"

**Comment by junalmeida** [comment]
@RobDaPraia yes, this means that changing that file manually is not an option.

**Comment by bjoveski** [comment]
Hey folks,

Thank you for developing Resumable, it's very well designed & documented and a pleasure to use.

Would you mind pushing the latest version (`1.1.2`) to the npm registry?

Thank you,
Bojan

**Comment by asmanp2012** [comment]
+
